<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PraticalArrayWidget</class>
 <widget class="QWidget" name="PraticalArrayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pre">
       <property name="font">
        <font>
         <family>文泉驿正黑</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>返回上级</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PraticalArrayDrawingWidget" name="widget" native="true">
     <property name="font">
      <font>
       <family>文泉驿正黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>文泉驿正黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>值</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="visitValue">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>文泉驿正黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="visitBtn">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>文泉驿正黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>值访问</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>文泉驿正黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>值</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="insertValue">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>文泉驿正黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>文泉驿正黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>索引</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="insertIndex">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>文泉驿正黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="insertBtn">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>文泉驿正黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>值插入</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>文泉驿正黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>索引</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="deleteIndex">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>文泉驿正黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="deleteBtn">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>文泉驿正黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>索引删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PraticalArrayDrawingWidget</class>
   <extends>QWidget</extends>
   <header>Ui/Drawing/praticalarraydrawingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
